### PR TITLE
Added DNSServer to API

### PR DESCRIPTION
**Pull request by mkurek**
Expose DNSServer in REST API (lack of it brokes network API).

### CODE REVIEW

File: src/ralph/dhcp/api.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing module docstring \[missing\-docstring\]

File: src/ralph/dhcp/api.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from ralph.api import RalphAPISerializer, RalphAPIViewSet, router
+from ralph.dhcp.models import DNSServer
+
+
+class DNSServerSerializer(RalphAPISerializer):
```
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too many ancestors \(11/7\) \[too\-many\-ancestors\]

File: src/ralph/dhcp/api.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    class Meta:
```
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too few public methods \(0/2\) \[too\-few\-public\-methods\]

File: src/ralph/dhcp/api.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        model = DNSServer
+        depth = 1
+
+
+class DNSServerViewSet(RalphAPIViewSet):
```
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too many ancestors \(16/7\) \[too\-many\-ancestors\]

File: src/ralph/dhcp/api.py (hunk 1 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing module docstring \[missing\-docstring\]

File: src/ralph/dhcp/api.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    queryset = DNSServer.objects.all()
```
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] ERROR: Class 'DNSServer' has no 'objects' member \[no\-member\]

File: src/ralph/dhcp/api.py (hunk 2 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

File: src/ralph/dhcp/api.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    serializer_class = DNSServerSerializer
+
+
+router.register(r'dns-servers', DNSServerViewSet)
+urlpatterns = []
```
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Invalid constant name "urlpatterns" \[invalid\-name\]

File: src/ralph/dhcp/api.py (hunk 2 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too many ancestors \(11/7\) \[too\-many\-ancestors\]

File: src/ralph/dhcp/api.py (hunk 3 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too few public methods \(0/2\) \[too\-few\-public\-methods\]

File: src/ralph/dhcp/api.py (hunk 4 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too many ancestors \(16/7\) \[too\-many\-ancestors\]

File: src/ralph/dhcp/api.py (hunk 5 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] ERROR: Class 'DNSServer' has no 'objects' member \[no\-member\]

File: src/ralph/dhcp/api.py (hunk 6 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Invalid constant name "urlpatterns" \[invalid\-name\]

File: src/ralph/dhcp/api.py (hunk 1 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing module docstring \[missing\-docstring\]

File: src/ralph/dhcp/api.py (hunk 2 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too many ancestors \(11/7\) \[too\-many\-ancestors\]

File: src/ralph/dhcp/api.py (hunk 3 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too few public methods \(0/2\) \[too\-few\-public\-methods\]

File: src/ralph/dhcp/api.py (hunk 4 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Too many ancestors \(16/7\) \[too\-many\-ancestors\]

File: src/ralph/dhcp/api.py (hunk 5 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] ERROR: Class 'DNSServer' has no 'objects' member \[no\-member\]

File: src/ralph/dhcp/api.py (hunk 6 above)
Comment (sputnikci) [inline src/ralph/dhcp/api.py]: \[Pylint\] INFO: Invalid constant name "urlpatterns" \[invalid\-name\]

**Comment by ar4s** [comment]
:ok:

**Pull request merged**